atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/TargetedSweeperTest.java

```diff
@@ -116,8 +122,8 @@ public static Object[] readBatchSize() {
     private TargetedSweepFollower mockFollower;
     private TimelockService timelockService;
     private PuncherStore puncherStore;
-    private boolean enabled = true;
-    private boolean enableAutoTuning = false;
+
+    private final AtomicReference<TargetedSweepRuntimeConfig> runtimeSupplier = new AtomicReference<>();
```

Comment:
FYSA: we generally use `Refreshable` to manage runtime configs specifically (since they can change at runtime, and consumers can subscribe to changes!). Refreshable also extends Supplier

It doesn't really apply in this case, since the consumer just takes a supplier of runtime config, but thought I'd let you know anyway